test/functions_test.py

```diff
@@ -181,14 +180,24 @@
 <BLANKLINE>
 Info: Nothing :)
 <BLANKLINE>
-[Warning] The ratio is not within the standard range. For `v60`, the ratio can be anywhere between `1/18` and `1/14`
 [Warning] The grind size is not within the standard range. For `v60`, the grind size can be anywhere between `400 um` and `700 um`
 >>> test_params = {"method":"custom", "cups":2, "coffee":6.0, "water":500, "coffee_ratio": 6, "water_ratio":1000, "info":"", "digits":3, "coffee_unit": "g"}
 >>> test_params = filter_params(test_params)
 >>> check_ratio_limits(test_params) == True
 True
 >>> check_grind_limits(test_params) == True
 True
+>>> test_params = {"method":"v60", "cups":2, "coffee":27.7, "water":500, "coffee_ratio": 1.2, "water_ratio":18.4, "info":"", "digits":3, "coffee_unit": "g", "water_unit": "g", "grind": 20}
+>>> test_params = filter_params(test_params)
+>>> check_ratio_limits(test_params) == True
```

Comment:
This should be False, right? because `1/18.4` is lower than lowest limit `1/18`

Reply:
```pycon
>>> r1 = 1.2 / 18.4
>>> r2 = 1 / 18
>>> r1
0.06521739130434782
>>> r2
0.05555555555555555
>>> r1 < r2
False
```